pkg/target/http.go

```diff
@@ -276,9 +277,30 @@ func createHTTPClient(oAuth2ClientID string, oAuth2ClientSecret string, oAuth2To
 
 // HTTPTargetConfigFunction creates HTTPTarget from HTTPTargetConfig
 func HTTPTargetConfigFunction(c *HTTPTargetConfig) (*HTTPTarget, error) {
+	var requestTimeoutInMillis int
+
+	if c.RequestTimeoutInMillis != 0 && c.RequestTimeoutInSeconds == 0 {
+		requestTimeoutInMillis = c.RequestTimeoutInMillis
+	}
+
+	if c.RequestTimeoutInMillis != 0 && c.RequestTimeoutInSeconds != 0 {
+		requestTimeoutInMillis = c.RequestTimeoutInMillis
+		log.Warn("Both 'request_timeout' and 'request_timeout_in_seconds' options are set. In this case 'request_timeout' takes precendence and 'request_timeout_in_seconds' is ignored. Using 'request_timeout_in_seconds' is deprecated, and will be removed in the next major version. Use 'request_timeout' only")
```

Comment:
Nit - warning refers to `request_timeout` which we renamed to `request_timeout_in_millis`